lib/posix_channel.ml

```diff
@@ -203,8 +203,9 @@ let receive protocols =
     finally
       (fun () ->
         Unix.connect s (Unix.ADDR_UNIX path);
-        let (_: int) = Unix.send s (Bytes.unsafe_of_string token) 0 (String.length token) [] in
-        let (_, _, fd) = Fd_send_recv.recv_fd s token 0 (String.length token) [] in
+        let token = Bytes.of_string token in
+        let (_: int) = Unix.send s token 0 (Bytes.length token) [] in
```

Comment:
We could alternatively do `Unix.send_substring` here and `Bytes.create (String.lenght token)` below, as the buffer is kind of throaway as it stands now

Reply:
That sounds even better to me!